src/app/private/board/page.tsx

```diff
@@ -0,0 +1,22 @@
+import Header from "@components/Header";
+import { Button, Typography } from "@mui/material";
+
+const Page = () => {
+  return (
+    <div>
+      <Header
+        title="Board"
+        rightIcon={
+          <Button style={{ textTransform: "none" }}>
```

Comment:
nit: is there a way to use tailwind for this? i tried adding a class and doesn't seem to work cause mui but maybe there's a workaround?